### ISSUE TITLE
installation: fix SQLAlchemy-Utils version

### ISSUE THREAD

**Issue by hachreak** [inveniosoftware/invenio#3039]
The master branch have a problem with a version conflict:

If you do: 
 - `pip install -e . --upgrade`
 - `python setup.py test`

It returns: 

```python

writing manifest file 'invenio.egg-info/SOURCES.txt'
running build_ext
Traceback (most recent call last):
  File "setup.py", line 352, in <module>
    'install_lib': _install_lib,
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/home/vagrant/.virtualenvs/invenio2/local/lib/python2.7/site-packages/setuptools/command/test.py", line 142, in run
    self.with_project_on_sys_path(self.run_tests)
  File "/home/vagrant/.virtualenvs/invenio2/local/lib/python2.7/site-packages/setuptools/command/test.py", line 121, in with_project_on_sys_path
    require('%s==%s' % (ei_cmd.egg_name, ei_cmd.egg_version))
  File "/home/vagrant/.virtualenvs/invenio2/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 940, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/vagrant/.virtualenvs/invenio2/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 832, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (SQLAlchemy-Utils 0.29.9 (/home/vagrant/.virtualenvs/invenio2/lib/python2.7/site-packages), Requirement.parse('SQLAlchemy-Utils>=0.30.0'), set(['wtforms-alchemy']))
```

**Comment by danmichaelo** [comment]
I also got this error with version 2.0.2

**Comment by jirikuncar** [comment]
@danmichaelo can you downgrade `wtforms-alchemy`? Please create separate issue for `maint-2.0` where we can discuss it. Thanks

**Comment by danmichaelo** [comment]
Thanks, @jirikuncar , seems like it was fixed in 83a9ba90 by requiring `wtforms-alchemy<0.13.1` , so the `main-2.0` branch is fine (I just tested). Just the 2.0.2 release is not. Is a new issue still needed then?

**Comment by jirikuncar** [comment]
@danmichaelo if it's already fixed in `maint-2.0` then it's not necessary. We will release 2.0.3 in upcoming days.
